macros/models/_sector/tokens/transfers_enrich.sql

```diff
@@ -0,0 +1,39 @@
+{%- macro case_when_token_standard(native_column, erc20_column, else_column) -%}
+CASE token_standard
+    WHEN 'native' THEN {{native_column}}
+    WHEN 'erc20' THEN {{erc20_column}}
+    WHEN 'bep20' THEN {{erc20_column}}
+    ELSE {{else_column}}
+END
+{%- endmacro-%}
+
+{%- macro transfers_enrich(blockchain, transfers_base) %}
+SELECT '{{blockchain}}' as blockchain
+, t.block_time
+, t.block_number
+, t.tx_hash
+, t.tx_index
+, t.evt_index
+, t.trace_address
+, t.token_standard
+, t.tx_from
+, t.tx_to
+, t."from"
+, t.to
+, t.contract_address
+, {{case_when_token_standard('\'SYMBOL\'', 'tokens_erc20.symbol', 'NULL')}} AS symbol
```

Comment:
am i reading this right, that the `'SYMBOL'` will be used directly? if so, what is the intention?

Reply:
This was just a placeholder, I'll update.